src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/AKVUnitTests.cs

```diff
@@ -21,31 +23,84 @@ public static class AKVUnitTests
         private const string cekCacheName = "_columnEncryptionKeyCache";
         private const string signatureVerificationResultCacheName = "_columnMasterKeyMetadataSignatureVerificationCache";
 
+        private static void ValidateAKVTraces(List<EventWrittenEventArgs> eventData, Guid threadActivityId)
+        {
+            Assert.NotNull(eventData);
+            Assert.NotEmpty(eventData);
+            int currentScope = 0;
+
+            // Validate event data captured.
+            Assert.All(eventData, item =>
+            {
+                Assert.Equal(DataTestUtility.AKVEventSourceName, item.EventSource.Name);
+                Assert.Equal(threadActivityId, item.ActivityId);
+                Assert.Equal(EventLevel.Informational, item.Level);
+                Assert.NotNull(item.Payload);
+                Assert.True(item.Payload.Count == 1);
```

Comment:
Can you use `Assert.Equal` instead?

Reply:
```suggestion
                Assert.Equal(1, item.Payload.Count);
```

Reply:
It didn't work quite well:

```
error xUnit2013: Do not use Assert.Equal() to check for collection size.
```

Reply:
```suggestion
                Assert.Single(item.Payload);
```
+ Use `Empty` if you don't expect any item.